assets/models/menu_backgrounds/MainMenuScene3.tscn

```diff
@@ -118,70 +123,507 @@ glow_hdr_threshold = 0.5
+
+[sub_resource type="GradientTexture1D" id="GradientTexture1D_egiip"]
+gradient = SubResource("Gradient_pi44u")
+
+[sub_resource type="ParticleProcessMaterial" id="ParticleProcessMaterial_cfpw8"]
+resource_local_to_scene = true
+emission_shape = 1
+emission_sphere_radius = 0.03
+angle_min = -179.0
+angle_max = 179.0
+direction = Vector3(0, 1, 0)
+spread = 1.0
+initial_velocity_min = 5.0
+initial_velocity_max = 5.0
+gravity = Vector3(0, -0.1, 0)
+color_ramp = SubResource("GradientTexture1D_egiip")
+alpha_curve = SubResource("CurveTexture_0f2id")
+turbulence_influence_min = 0.0
+turbulence_initial_displacement_max = 0.2
+
+[sub_resource type="QuadMesh" id="QuadMesh_xviu6"]
+resource_local_to_scene = true
+size = Vector2(2, 2)
+
+[sub_resource type="StandardMaterial3D" id="StandardMaterial3D_nkjo6"]
+resource_local_to_scene = true
 transparency = 4
-cull_mode = 2
 vertex_color_use_as_albedo = true
-vertex_color_is_srgb = true
-albedo_texture = ExtResource("2")
+albedo_texture = ExtResource("3_ooioo")
 metallic = 0.18
 metallic_specular = 1.0
 roughness = 0.7
 normal_enabled = true
-normal_scale = 6.19
-normal_texture = ExtResource("10")
+normal_scale = 6.2
+normal_texture = ExtResource("4_nq5x3")
+disable_receive_shadows = true
 billboard_mode = 3
 particles_anim_h_frames = 1
 particles_anim_v_frames = 1
 particles_anim_loop = false
 
-[sub_resource type="QuadMesh" id="26"]
-material = SubResource("25")
+[sub_resource type="Curve" id="Curve_yggn1"]
+resource_local_to_scene = true
+_data = [Vector2(0.00344828, 1), 0.0, 0.0, 0, 0, Vector2(0.544828, 0.438202), -3.2615, -3.2615, 0, 0, Vector2(0.996552, 0), 0.0, 0.0, 0, 0]
+point_count = 3
+
+[sub_resource type="CurveTexture" id="CurveTexture_83pdj"]
+resource_local_to_scene = true
+curve = SubResource("Curve_yggn1")
+
+[sub_resource type="ParticleProcessMaterial" id="ParticleProcessMaterial_jsyrn"]
+resource_local_to_scene = true
+emission_shape = 1
+emission_sphere_radius = 0.03
+angle_min = -179.0
+angle_max = 179.0
+direction = Vector3(0, 1, 0)
+spread = 1.0
+initial_velocity_min = 5.0
+initial_velocity_max = 5.0
+gravity = Vector3(0, -0.1, 0)
+color_ramp = SubResource("GradientTexture1D_egiip")
+alpha_curve = SubResource("CurveTexture_83pdj")
+turbulence_influence_min = 0.0
+turbulence_initial_displacement_max = 0.2
+
+[sub_resource type="QuadMesh" id="QuadMesh_rynfs"]
+resource_local_to_scene = true
+size = Vector2(3, 3)
+
+[sub_resource type="StandardMaterial3D" id="StandardMaterial3D_61jtu"]
+resource_local_to_scene = true
+transparency = 4
+vertex_color_use_as_albedo = true
+albedo_texture = ExtResource("3_ooioo")
+metallic = 0.18
+metallic_specular = 1.0
+roughness = 0.7
+normal_enabled = true
+normal_scale = 6.2
+normal_texture = ExtResource("4_nq5x3")
+disable_receive_shadows = true
+billboard_mode = 3
+particles_anim_h_frames = 1
+particles_anim_v_frames = 1
+particles_anim_loop = false
+
+[sub_resource type="Curve" id="Curve_qwc45"]
+resource_local_to_scene = true
```

Comment:
It seems a bit like you are overusing the local to scene checkbox. As far as I can tell these resources are only in this scene so checking the checkbox doesn't do anything, and it should only be used for resources that are *modified* during runtime. Otherwise it basically has no effect so it is an unnecessary non-default value that is set.

For unsharing resources set in the editor on multiple objects, you need to right click the resource and click "make unique"
As you haven't written any C# code to interact with these resources there's actually nothing that would modify them during runtime so `resource_local_to_scene = true` does nothing useful, and I think it might cause some de-optimization in the engine so I'd prefer these unnecessary usages of these to be removed.